lib/src/boom_menu.dart

```diff
@@ -17,11 +17,17 @@ class BoomMenu extends StatefulWidget {
   final Color foregroundColor;
   final double elevation;
   final ShapeBorder fabMenuBorder;
+  final Alignment fabAlignment;
 
   final double marginLeft;
   final double marginRight;
   final double marginBottom;
```

Comment:
Remove those lines

Reply:
Sorry, was making this pull request not from IDE, I think we probably should leave those and add them back to Positioned(), our changes don't break user's code.

Reply:
I ll make another one with changes today